mozilla-b2g/gaia
# Bug 881144 - hdpi/xhdpi resolution assets and layout for E-mail app

Support hdpi resolution device.
http://bugzil.la/881144

To test this patch on HVGA/WVGA devices:
$ make install-gaia APP=email GAIA_DEV_PIXELS_PER_PX=1.5

What "GAIA_DEV_PIXELS_PER_PX" do is to replace image files by @1.5x assets when exist.
Images will be too large (and break the layout) if we missed to set background-size for them in the patch.
If this patch works well, we will get correct layout on both HVGA and WVGA (and ideally qHD, but AFAIK we don't have to confirm it at this time) screen.

To test on nightly is also possible:
1. $make DEBUG=1 GAIA_DEV_PIXELS_PER_PX=1.5
2. enter "about:config" and set "layout.css.devPixelsPerPx" as "1.5"
3. set responsive design window to 320 x 533 px (which is CSS pixel dimension for 480 x 800 device pixel dimension).
